pkgs/applications/networking/browsers/firefox/packages.nix

```diff
@@ -7,10 +7,10 @@ in
 rec {
   firefox = common rec {
     pname = "firefox";
-    ffversion = "88.0";
+    ffversion = "88.0.1";
     src = fetchurl {
       url = "mirror://mozilla/firefox/releases/${ffversion}/source/firefox-${ffversion}.source.tar.xz";
-      sha512 = "f58f44f2f0d0f54eae5ab4fa439205feb8b9209b1bf2ea2ae0c9691e9e583bae2cbd4033edb5bdf4e37eda5b95fca688499bed000fe26ced8ff4bbc49347ce31";
+      sha512 = "0sakkl4bsjcl7bq08dgvmqjpcz2i5h7bmbf4miv61x25zg50g3k5rppcmw5p7f4hl5z9dx36dng3z0qcffpj7gfhdf257x41fazrmz2";
```

Comment:
Thanks!

Reply:
It is generated by `update.nix`, which relies on generic `update-source-version` to guess the hash. We should update `update.nix` to use `SHA512SUMS`, perhaps in separate PR.

Reply:
Created #121761.

Reply:
Updated the hash.